### PR TITLE
Prepare support for python 3.8

### LINKED ISSUE #701: Python3.8 compatibility
python3.8 comes with a number of smaller API breaks that we need to account for, lest they become bugs as soon as people try running panel with the new version.

### PR DESCRIPTION
**Pull request by a-recknagel**
closes #701

### CODE REVIEW

**Comment by philippjfr** [comment]
Thanks @a-recknagel, much appreciated!

File: panel/util.py [hunk 1]
```diff
@@ -9,9 +9,13 @@
...
 import datetime as dt
 
-from collections import defaultdict, MutableSequence, MutableMapping, OrderedDict
 from datetime import datetime
 from six import string_types
+from collections import defaultdict, OrderedDict
+try:
+    from collections.abc import MutableSequence, MutableMapping
+except ImportError:  # support for python>3.8
+    from collections import MutableSequence, MutableMapping
```
Comment (jbednar) [inline panel/util.py]: Isn't this comment backwards?  I.e., isn't the `try` clause (collections.abc) required for python>3.8, and the `except` clause is for some older version (<3.3)?

Comment (a-recknagel) [reply panel/util.py]: yeah, I put the comparison the wrong way round, should have been `python<3.8`

Comment (a-recknagel) [reply panel/util.py]: oh, it was already merged. I'll include it in the docs PR

Comment (jbednar) [reply panel/util.py]: Ok, but it's not true the other way around either; the `except` isn't needed for python<3.8, only for python<3.3.

Comment (a-recknagel) [reply panel/util.py]: Ah, thanks for pointing that out.

**Pull request merged**